examples/fitness/lib/settings.dart

```diff
@@ -112,19 +112,17 @@ class SettingsFragment extends StatefulComponent {
           child: new BlockBody([
             new DrawerItem(
               onPressed: () { _handleBackupChanged(!(userData.backupMode == BackupMode.enabled)); },
-              children: [
+              child: new Flex([
                 new Flexible(child: new Text('Back up data to the cloud')),
                 new Switch(value: userData.backupMode == BackupMode.enabled, onChanged: _handleBackupChanged)
-              ]
+              ], direction: FlexDirection.horizontal)
             ),
             new DrawerItem(
               onPressed: () => _handleGoalWeightPressed(),
-              children: [
-                new Flex([
-                  new Text('Goal Weight'),
-                  new Text(goalWeightText, style: Theme.of(this).text.caption),
-                ], direction: FlexDirection.vertical, alignItems: FlexAlignItems.start)
-              ]
+              child: new Flex([
+                new Text('Goal Weight'),
+                new Text(goalWeightText, style: Theme.of(this).text.caption),
+              ], direction: FlexDirection.vertical, alignItems: FlexAlignItems.start)
```

Comment:
wrap these one to a line for consistency with the others. What I've been doing to get the indentation right in these cases is have the three lines after the [ indented by 4 instead of 2.